### DISCUSSION TITLE
Problem connecting to SSH from GitHub

### DISCUSSION THREAD

**Discussion by ghost** [community/community#40198] [Apps, API and Webhooks] [answered]
### Select Topic Area

Bug

### Body

root@zorin:~/.ssh# ssh-keygen -t ed25519 -C "feliphe@tuta.io"
Generating public/private ed25519 key pair.
Enter file in which to save the key (/root/.ssh/id_ed25519):  
/root/.ssh/id_ed25519 already exists.
Overwrite (y/n)? y
Enter passphrase (empty for no passphrase): 
Enter same passphrase again: 
Your identification has been saved in /root/.ssh/id_ed25519
Your public key has been saved in /root/.ssh/id_ed25519.pub
The key fingerprint is:
SHA256:F7fKXgOH+Kq8is8tekgzX7D6L/AODJvYTdFfki7u4L4 feliphe@tuta.io
The key's randomart image is:
+--[ED25519 256]--+
|                 |
|     .   .       |
|    . . o o .    |
|    .. o + + .   |
|.   .o. S + o    |
|.*=o.... + +     |
|+.+Oo..   + o    |
|  o=*=   o . .   |
|  oBE*Bo. .      |
+----[SHA256]-----+
root@zorin:~/.ssh# ssh-add -l -E md5
Could not open a connection to your authentication agent.
**NOT CONNECTING FOR SSH FROM GITHUB**
root@zorin:/home/zorin/GitHub# ssh -T email-explo@github.com
email-explo@github.com: Permission denied (publickey).

**Comment by airtower-luna** [comment, marked as answer]
I see two problems there:

1. To connect to GitHub over SSH you have to connect to `git@github.com`. Any other username (`email-explo` in your case) won't work.
2. You're working as root. SSH should work anyway, but it's bad practice because it's an unnecessary security risk.
